config/samples/infoblox/operator_v1beta1_infoblox_openshift.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: externaldns.olm.openshift.io/v1beta1
```

Comment:
I see samples added for AWS, GCP, Azure and Infoblox but not Bluecat.. Is there a reason for that or just missed?

Reply:
Added BlueCat samples too